.changeset/cold-numbers-push/changes.json

```diff
@@ -0,0 +1,69 @@
+{
+  "releases": [
+    { "name": "@keystone-alpha/auth-passport", "type": "major" },
+    { "name": "@keystone-alpha/keystone", "type": "major" },
```

Comment:
Is this really a major bump to `@keystone-alpha/keystone`?

Reply:
Actually no, only the other one is because you can't import `PasswordAuthStrategy` from keystone anymore. But now that I think about it, I don't think this has any breaking change for Keystone itself.